src/CriteriaHandler.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Data\Db;
+
+use LogicException;
+use Yiisoft\Data\Db\FilterHandler\AllHandler;
+use Yiisoft\Data\Db\FilterHandler\AnyHandler;
+use Yiisoft\Data\Db\FilterHandler\BetweenHandler;
+use Yiisoft\Data\Db\FilterHandler\Context;
+use Yiisoft\Data\Db\FilterHandler\EqualsEmptyHandler;
+use Yiisoft\Data\Db\FilterHandler\EqualsHandler;
+use Yiisoft\Data\Db\FilterHandler\EqualsNullHandler;
+use Yiisoft\Data\Db\FilterHandler\ExistsHandler;
+use Yiisoft\Data\Db\FilterHandler\GreaterThanHandler;
+use Yiisoft\Data\Db\FilterHandler\GreaterThanOrEqualHandler;
+use Yiisoft\Data\Db\FilterHandler\InHandler;
+use Yiisoft\Data\Db\FilterHandler\LessThanHandler;
+use Yiisoft\Data\Db\FilterHandler\LessThanOrEqualHandler;
+use Yiisoft\Data\Db\FilterHandler\LikeHandler;
+use Yiisoft\Data\Db\FilterHandler\NotHandler;
+use Yiisoft\Data\Db\FilterHandler\QueryHandlerInterface;
+use Yiisoft\Data\Reader\FilterHandlerInterface;
+
+final class CriteriaHandler
```

Comment:
Need phpdoc w/ description about the purpose of the class.

Reply:
done